hnn_core/parallel_backends.py

```diff
@@ -337,10 +337,6 @@ def simulate(self, net):
             The Dipole results from each simulation trial
         """
 
-        # just use the joblib backend for a single core
-        if self.n_procs == 1:
-            return JoblibBackend(n_jobs=1).simulate(net)
```

Comment:
not quite sure why this was necessary?

Reply:
JoblibBackend for 1 job is essentially no backend. Don't bother creating the mpi child process for a single processor.